lib/ddtrace/profiling/event.rb

```diff
@@ -0,0 +1,13 @@
+module Datadog
+  module Profiling
+    # Describes a sample of some data obtained from the runtime.
+    class Event
+      attr_reader \
+        :timestamp
+
+      def initialize
+        @timestamp = Time.now.utc.to_i
```

Comment:
IIUC this stores an integer? I think you can keep the `Time` object around, it might be easier to manipulate — you can convert it to an integer as needed when it'll be serialized.

Reply:
There was no spec for how events are structured so I just threw this in as is because it takes less memory. I'll wait till we have more specific details about events, and establish the structure changes all at once.